sofascoreAcademy/sofascoreAcademy/Views/CustomNavigationTitleView.swift

```diff
@@ -0,0 +1,59 @@
+//
+//  CustomNavigationTitleView.swift
+//  sofascoreAcademy
+//
+//  Created by Tina Jureško on 03.04.2025..
+//
+
+import Foundation
+import UIKit
+import SnapKit
+import SofaAcademic
+
+class CustomNavigationTitleView: BaseView {
+    private let leagueLogoImageView = AsyncImageView()
+    private let titleLabel = UILabel()
+    
+    override func addViews() {
+        super.addViews()
```

Comment:
Sigurno si primjetila da znaš imati dosta linija koje ti ne trebaju nužno u kodu. Probaj proći po kodu nak nešto implementiraš vidjeti što ti se duplicira ili je reduntantno i to čistiti. Maknuti liniju vidjeti jel radi, pa vidjeti zašto radi bez toga...